castra/core.py

```diff
@@ -372,6 +377,17 @@ def _decategorize(categories, df):
     return extra, new_categories, new_df
 
 
+def make_categorical(s, categories):
+    if s.name in categories:
+        idx = pd.Index(categories[s.name], tupleize_cols=False, dtype='object')
+        idx.is_unique = True
+        cat = pd.Categorical(s.values, categories=idx, fastpath=True,
+                             ordered=False)
```

Comment:
i'm not sure if ordered should always be `False` here. in the long run i don't think it should